### ISSUE TITLE
Add security level check

### ISSUE THREAD

**Issue by maks-a** [antelle/keeweb#49]
It would be nice to add one button for security level check. This button checks all the open files for weak, reused and expanded passwords.
Maybe also to add result table with sortable columns: entries, pwd strength, last updated time.

**Comment by antelle** [comment]
Yeah, audit feature is planned for future releases and will be available in advanced search. It will be able to show weak, duplicated, old and expired passwords.

**Comment by KaKi87** [comment]
Hello, can password quality warnings be disabled ? Thanks

**Comment by antelle** [comment]
Yes, in settings. When you click the X button in the warning, it will open the settings page.
